index.html

```diff
@@ -11,7 +11,36 @@
 
 <body>
 
-   <!-- The pop-up section -->
+    <section class="the-pop-up" id="poped">
+        <span id="x-mark"> <i class="fa-solid fa-x" ></i> </span>
+     <h2 class="p-topic">Multi-Post Stories</h2>
+     <img class="snapshoot"  src="images/pop-up.png" alt="pop-up snapshoot">
+     <p class="description">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.
+     </p>
+     <div class="skilling">
+     <ul class="skiil" id="ski">
+         <li class="vecto">.</li>
+         <li class="skill">html</li>
+         <li class="vecto">.</li>
+         <li class="skill">css</li>
+         <li class="vecto">.</li>
+         <li class="skill">ruby on rails</li>
+         <li class="vecto">.</li>
+      </ul>
+    </div>
+     <div class="btncontainer">
+         <button class="bt1"  type="button"> 
+             See live<span> <i class="fa-solid fa-up-right-from-square"></i> </span>
+         </button>
+      <button  class="bt2" type="button">
+           See Project
+            <i class="fa-brands fa-github"></i></button>
+      </div>
+ </section>
```

Comment:
- [ ] You have done an amazing job styling your popup for both desktop and mobile views :clap: I have however noticed that the content displayed for the popup is static and is hardcoded. Kindly note that according to the project requirements, you are supposed to generate the contents of the popup dynamically from your Array. Currently, this is not the case making your objects "hanging" as they are not being used. Kindly ensure that you generate the contents of the popup from your array and are not hardcoded.

![image](https://user-images.githubusercontent.com/86318284/180205195-f6285cea-b1a8-4ed7-9e12-25bc8674aba9.png)

- [ ] I have also realized that your desktop version of the modal popup does not match the [chosen Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10). Kindly make sure that you fix this so that you meet all the project requirements.

**This is how your modal popup looks on my end:**

![image](https://user-images.githubusercontent.com/86318284/180206071-c5cfd46e-45b0-4637-bbb4-13c05dab9395.png)

**This is the desired Figma design modal popup:**

![image](https://user-images.githubusercontent.com/86318284/180206216-b46ab698-9139-492b-b24c-63d3135680a5.png)

Reply:
All DONE and FIXED